Gemfile.lock

```diff
@@ -401,5 +401,8 @@ DEPENDENCIES
   web-console (= 2.1.3)
   whenever
 
+RUBY VERSION
+   ruby 2.3.1p112
```

Comment:
How is this ruby version thing? I need it to run this PR in my machine... we better fix it asap so that this branch can be merged without this change.